src/components/KonnectorTile.jsx

```diff
@@ -171,7 +177,8 @@ export const KonnectorTile = ({ konnector, isInMaintenance, loading }) => {
   const errorToDisplay = !userError && userError !== null ? userError : error
 
   const handleClick = event => {
-    if (status === STATUS.NO_ACCOUNT) {
+    // Navigate to the store if the konnector is a suggestion (ie not installed, not connected)
+    if (isSuggestion) {
```

Comment:
Why did the status stop working? Be careful, because the KonnectorTile is also used in Services which you didn't modify. Is this intentional?